streams/src/main/java/org/apache/kafka/streams/kstream/internals/foreignkeyjoin/CombinedKeySchema.java

```diff
@@ -60,7 +60,7 @@ public void init(final ProcessorContext<?, ?> context) {
         foreignKeyDeserializer = foreignKeyDeserializer == null ? (Deserializer<KO>) context.keySerde().deserializer() : foreignKeyDeserializer;
     }
 
-    Bytes toBytes(final KO foreignKey, final K primaryKey) {
+    public Bytes toBytes(final KO foreignKey, final K primaryKey) {
```

Comment:
I guess we don't need to change this

Reply:
Oh yes you're right, I was using it locally as we emulate fk joins for some republish process.

**Is it possible to keep it public** to allow users to serialize a fk join subscription key, without copy/pasting the full method on the user side ?

### Explanation 
On our side, we have a generic state api that allows us to query any stores, including the internal stores like subscription stores, but we need to serialize the key before, as the store serializer is a BytesSerializer and not a CombinedKey serializer (so we cannot just pass a CombinedKey). We are also able to query all the subscribed keys thanks to `CombinedKeySchema.prefixBytes` + `prefixScan`.

Reply:
```suggestion
    Bytes toBytes(final KO foreignKey, final K primaryKey) {
```